src/include/optimizer/rules/unnesting_rules.h

```diff
@@ -48,14 +48,49 @@ class UnnestMarkJoinToInnerJoin : public Rule {
 };
 
 /**
- * Unnest Single Join to Inner Join
+ * Transform Single Join to Inner Join
```

Comment:
Should also update the TODO at the start of the file as we are now transforming Single Joins